src/redux/wallets.js

```diff
@@ -224,6 +227,56 @@ export const createAccountForWallet = (id, color, name) => async (
   return newWallets;
 };
 
+export const getWalletENSAvatars = async (walletsState, dispatch) => {
+  const { wallets, walletNames, selected } = walletsState;
+  const walletKeys = Object.keys(wallets);
+  let updatedWallets;
+  for (const key of walletKeys) {
```

Comment:
we don't care to collapse these into a promse.all ?

Reply:
oh true